conf/local.conf.sample

```diff
@@ -48,275 +46,83 @@ MACHINE ??= "qemux86"
 #SOURCE_MIRROR_URL = "file:///cache/downloads"
 #SOURCE_MIRROR_URL = "http://myserver/sources"
 
-# Our download directory / cache for the workspace
-DL_DIR ?= "${FLEXDIR}/downloads"
-
-# Continue to pull downloads from the build dir if available
-PREMIRRORS:prepend = "\
-    .*://.*/.* file://${FLEXDIR}/${MACHINE}/downloads/ \n \
-    .*://.*/.* file://${TOPDIR}/downloads/ \n \
-"
-
-# BitBake has the capability to accelerate builds based on previously built
-# output. This is done using "shared state" files which can be thought of as
-# cache objects and this variable determines where those files are placed.
-SSTATE_DIR ?= "${TOPDIR}/cached-binaries"
-
-# Continue to pull cached binaries from the Sokol Flex OS install
-SSTATE_MIRROR_SITES += "\
-    file://${FLEXDIR}/${MACHINE}/cached-binaries \
-    file://${FLEXDIR}/cached-binaries \
-"
-
-# Where to place the build output and temporary artifacts
-#
-# The default value follows:
-#TMPDIR = "${TOPDIR}/tmp"
-
-# When we're using an external toolchain, we don't want to ship a newly built
-# toolchain inside the Yocto SDK.
-include conf/include/drop-toolchain-from-sdk.inc
-
-#
-# Package Management configuration
-#
-# This variable lists which packaging formats to enable. Multiple package backends
-# can be enabled at once and the first item listed in the variable will be used
-# to generate the root filesystems.
-#
-#  - 'package_deb' for debian style deb files
-#  - 'package_ipk' for ipk files are used by opkg (a debian style embedded package manager)
-#  - 'package_rpm' for rpm style packages
-#
-# We default to ipk:
-#PACKAGE_CLASSES ?= "package_ipk"
+# Shared-state files from other locations
+#SSTATE_MIRROR_SITES += "http://mymirror/sstate"
+#SSTATE_MIRROR_SITES += "\
+#    http://someserver.tld/share/sstate \
+#    file:///some/local/dir \
+#"
 
-# SDK target architecture
-#
-# Set to one of the mingw32 SDKMACHINEs to target Windows rather than Linux.
-# Warning: if you choose to add packages to TOOLCHAIN_HOST_TASK and target
-# Windows, then you must add the meta-mingw layer to your configuration.
+# Specify the architecture/os where a built SDK will be installed
+# Set to one of the mingw32 SDKMACHINEs to target Windows rather than Linux
 #
 # Valid values: i686, x86_64, i686-mingw32, x86_64-mingw32
 #SDKMACHINE ?= "x86_64"
 
-# Uncomment to make populate_sdk write a tar file rather than a .sh installer
-#SDK_PACKAGING_FUNC = ""
-
-# By default, only include the target sysroot in the SDK, not host tools.
-# Comment this line to change that. This packagegroup includes host tools like
-# autoconf, automake, etc.
+# By default, include a toolchain the SDK, but not host tools like autoconf
 TOOLCHAIN_HOST_REMOVE ??= "nativesdk-packagegroup-sdk-host"
 TOOLCHAIN_HOST_TASK:remove = "${TOOLCHAIN_HOST_REMOVE}"
 
-# TOOLCHAIN_HOST_TASK is used to add host packages to the SDK, for
-# example, to add bash:
+# Add host packages to the SDK
 #TOOLCHAIN_HOST_TASK_EXTRA += " nativesdk-bash"
```

Comment:
Not related but why do we need a space here with += ?

Reply:
BTW I think customers are not using this. Should we dropt this as well?

Reply:
I think we need marketing input on this, as I think wanting to add  extra packages into an SDK would be common, such as host tooling for development, things like qmake for qt (though that particular case I'd hope it'd be automatic for images including qt6).